dist/routable-component.js

```diff
@@ -144,7 +144,7 @@ var routableComponent =
 	          var name = _step.value;
 	
 	          var component = components[name];
-	          this.register_component(name, component);
+	          this.registerComponent(name, component);
```

Comment:
__[eslint]__
* Error - Mixed spaces and tabs. (no-mixed-spaces-and-tabs)
* Error - Expected indentation of 2 space characters but found 0. (indent)

<!-- Comment by [SideCI](https://sideci.com) -->